src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
```

Comment:
@mcovarr and I talked about this earlier. We agreed that for the time being we'll not adhere to a fixed column width requirement but play it by ear in terms of chained operations & line length. If it's reading "like english" longer is AOK but if it's starting to really look like multiple expressions it should be a sign to split it up. We were going to see how often we agreed/disagreed.

re official style guide, there is no official guidance. I poked around on the more prominent style guides last night and the vast majority were >= 120 characters. I actually always thought the horizontal line that intellij draws was 80 chars but at some point they greatly expanded that, my thinking that that was 80 chars drove my hefty objection to miguel's "too long" response :)